### PR TITLE
remove classifications flag TM-3066

### PR DESCRIPTION
**Pull request by SidelineCory24**
**Feature Flag Part 3 Removal**
[Config File Part 3 Changes PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2197)

When the `classifications` flag is set to false, the `Edit Classifications` button should show when viewing a client's public profile.

image

### CODE REVIEW

File: src/Components/ProfileDashboard/Classifications/Classifications.jsx [hunk 1]
```diff
@@ -57,9 +56,6 @@ const Classifications = props => {
     }
   };
 
-  const useClassificationsEditor = () => checkFlag('flags.classifications');
-  const displayClassificationsEditor = useClassificationsEditor() && isPublic;
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/Classifications/Classifications.jsx:61]: We want to keep the `isPublic` check, cause we don't want Jenny editing her own Classifications lol
image

File: src/Components/ProfileDashboard/Classifications/Classifications.jsx [hunk 2]
```diff
@@ -83,7 +79,7 @@ const Classifications = props => {
         </div>
       </div>
       {
-        !editView && displayClassificationsEditor &&
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/Classifications/Classifications.jsx:86]: you can just put the isPublic here, no need to have it in another variable

File: src/Components/ProfileDashboard/Classifications/Classifications.jsx (hunk 1 above)
Comment (SidelineCory24) [reply src/Components/ProfileDashboard/Classifications/Classifications.jsx:61]: resume inflation 😎

File: src/Components/ProfileDashboard/Classifications/Classifications.jsx (hunk 2 above)
Comment (SidelineCory24) [reply src/Components/ProfileDashboard/Classifications/Classifications.jsx:86]: done

**Review by elizabeth-jimenez — APPROVED** [review]

**Pull request merged**